docs/samples/samples.json

```diff
@@ -6,20 +6,35 @@
             "name": "hello-k8s",
             "path": "k8s/hello-world",
             "artifacts": [
+                {
+                    "file": "target-container.yaml",
+                    "type": "target-container",
+                    "name": "samplek8starget"
+                },
                 {
                     "file": "target.yaml",
                     "type": "target",
-                    "name": "sample-k8s-target"
+                    "name": "samplek8starget-v1"
                 },
+                {
+                    "file": "solution-container.yaml",
+                    "type": "solution-container",
+                    "name": "sampleprometheusserver" 
+                }, 
                 {
                     "file": "solution.yaml",
                     "type": "solution",
-                    "name": "sample-prometheus-server" 
+                    "name": "sampleprometheusserver-v1" 
                 }, 
+                {
+                    "file": "instance-container.yaml",
```

Comment:
Do we need instance-container.yaml? Before @LeiQL 's change to remove instance container.

Reply:
If this PR is merged before #323, will update my PR to remove the sample reference.